package.json

```diff
@@ -1,34 +1,34 @@
 {
-  "name": "phishtray",
-  "description": "Wrapper for frontend to allow Heroku builds.",
-  "version": "1.0.0",
-  "dependencies": {
-    "carbon-components": "^9.1.1",
-    "carbon-components-react": "^6.6.3",
-    "carbon-icons": "^7.0.7",
-    "date-fns": "^1.29.0",
-    "emotion": "^9.2.4",
-    "immer": "^1.3.1",
-    "react": "^16.4.1",
-    "react-circular-progressbar": "^1.0.0",
-    "react-dom": "^16.4.1",
-    "react-emotion": "^9.2.4",
-    "react-markdown": "^3.6.0",
-    "react-redux": "^5.0.7",
-    "react-remarkable": "^1.1.3",
-    "react-router-dom": "^4.3.1",
-    "react-scripts": "1.1.4",
-    "redux": "^4.0.0",
-    "redux-logger": "^3.0.6",
-    "redux-persist": "^5.10.0",
-    "redux-thunk": "^2.3.0",
-    "reselect": "^3.0.1"
-  },
-  "scripts": {
-    "heroku-prebuild": "cp frontend/yarn.lock ./",
-    "heroku-postbuild": "cd frontend && yarn build"
-  },
-  "engines": {
-    "yarn": "1.12.3"
-  }
+    "name": "phishtray",
+    "description": "Wrapper for frontend to allow Heroku builds.",
+    "version": "1.0.0",
```

Comment:
It's worth bumping the version if you change the requirements.

Reply:
done